src/components/open-source-item.js

```diff
@@ -0,0 +1,50 @@
+import { Paper } from "material-ui";
+import colors from "../styles";
+
+class OpenSourceItem extends React.Component {
+	constructor() {
+		super();
```

Comment:
@gfogle did some research on this only need to call `super(props)` if you need to access props in the constructor. react adds props to the instance internally after constructor.